extensions/wikia/WikiaPhotoGallery/WikiaPhotoGallery.class.php

```diff
@@ -118,6 +118,21 @@ function __construct() {
 			'hideoverflow' => false,
 			'sliderbar' => array('bottom','left')
 		);
+
+		$this->mAvalaibleUniqueParams = array_values(
```

Comment:
It's misspelled, should be `mAvailableUniqueParams`. And yes, `mAvalaibleGalleryParams` is misspelled too.